### ISSUE TITLE
Lock actions improvement

### ISSUE THREAD

**Issue by COMMONWARE** [heggink/domoticz-nuki#6]
Thanks for the great effort. The plugin works fine with a single bridge and 2 locks.

A nice improvement would be to implement all the lock actions
https://developer.nuki.io/page/nuki-bridge-http-api-1-10/4/#heading--lock-actions

I will try to implement this and create a pull request

**Comment by heggink** [comment]
Agree with a but: The default lock device in domoticz does not know about advanced lock actions (such as lock and go) which means that you have to implement a custom device. That then won't be a lock anymore. This is why I implemented the unlatch as a separate device. What we could do is to make this a custom device with the additional lock actions rather than only the unlatch.

**Comment by COMMONWARE** [comment]
Yes, I just read about it. I wasn't aware of the limitations of the device type when I wrote the issue. After reading the limitations, the first thing that came to my mind was creating a custom device to unlatch. 
I'm going to look into the creation of a custom device.

**Comment by heggink** [comment]
Cool! please note that I just merged the feature-unlatch branch into master.
